### PR TITLE
[7.x] Basic tax engine shipping VAT fix

### PR DESCRIPTION
**Pull request by rolinbos**
Add this moment when you define that the basic tax engine ```included_in_prices``` is set to true, it will think that the shipping costs also is included in the price. But the calculation of it isn't correct.
With this PR, it will check for the basic tax engine if the ```included_in_prices``` is true or false. 
Before it calculates the VAT for shipping with 100 but it needs to be 100 + VAT rate so that you can calculate the price without VAT.

For example the shipping price is 6.50 with VAT. And the VAT is 21%

Before this PR:

```650 / 100 * 21 = 136,5 = 137```

So the price without VAT is:

```650 - 137 = 513```

But this isn't right.

With this PR:

```650 / 121 * 21 = 112,80 = 113```

So the price without VAT is:

```650 - 113 = 537```

### CODE REVIEW

File: resources/blueprints/collections/orders/order.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+tabs:
```
Comment (duncanmcclean) [inline resources/blueprints/collections/orders/order.yaml]: The changes in this file seem unrelated. Can you please revert them?

File: resources/blueprints/collections/orders/orders.yaml [hunk 2]
```diff
@@ -1,372 +1,3 @@
-title: Order
 tabs:
```
Comment (duncanmcclean) [inline resources/blueprints/collections/orders/orders.yaml]: The changes in this file seem unrelated. Can you please revert them?

File: resources/blueprints/collections/products/product.yaml [hunk 3]
```diff
@@ -8,20 +8,30 @@ tabs:
             field:
               type: text
               required: true
-              width: 50
           -
-            handle: price
+            handle: product_variants
```
Comment (duncanmcclean) [inline resources/blueprints/collections/products/product.yaml]: We don't need the `product_variants` field in this blueprint. Are you able to revert this file back to how it was?

**Review by duncanmcclean — CHANGES_REQUESTED** [review]
There seems to be a couple unrelated changes in this PR (primarily around the blueprints). Can you please revert them?

**Comment by rolinbos** [comment]
Hi @duncanmcclean, thanks for your feedback. I've changed the files.

**Review by duncanmcclean — APPROVED** [review]

**Comment by duncanmcclean** [comment]
Thanks!
